circuit_knitting_toolbox/circuit_cutting/cutting_evaluation.py

```diff
@@ -77,8 +203,240 @@ def append_measurement_circuit(
         actual_qubit = qubit_locations[subqubit]
         if genobs_x[subqubit]:
             if genobs_z[subqubit]:
-                qc.sdg(actual_qubit)
+                qc.sdg(actual_qubit)  # pragma: no coverage
```

Comment:
@garrison not urgent, but do you have an immediate idea of how to trigger this? Can we add something to `test_cutting_roundtrip`?

Reply:
You need an observable with a Pauli `Y` in it somewhere.

Reply:
Oh wait.  There _is_ such an observable.  Hmm.

Reply:
The roundtrip observable is "ZZZ". Let me add a Y

Reply:
There were such observables in the roundtrip tests, but it looks like you eliminated them.  I chose those observables to have a good mix of I,X,Y,Z and different phase coefficients.

Reply:
Fixed